### PR TITLE
Vt bug fix

### PR DESCRIPTION
**Pull request by roysagi**
## Contributing to Cortex XSOAR Content
Make sure to register your contribution by filling the [contribution registration form](https://forms.gle/XDfxU4E61ZwEESSMA)

**The Pull Request will be reviewed only after the contribution registration form is filled.**

## Status
- [x] In Progress
- [x] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: https://github.com/demisto/etc/issues/30006

## Description
Added *ResponseContentSHA256* and *ResponseHeaders* to context output for the **vt-private-get-url-report** command in case the *allInfo* argument is set.

### CODE REVIEW

File: Packs/VirusTotal-Private_API/Integrations/VirusTotal-Private_API/VirusTotal-Private_API.py [hunk 1]
```diff
@@ -705,17 +720,18 @@ def create_url_report_output(url, response, threshold, max_len, short_format):
...
-        if ec_url:
-            ec_url['VirusTotal'] = {
-                'Resolutions': resolution[:max_len]
-            }
-        else:
-            ec_url.update({
-                'VirusTotal': {'Resolutions': resolution[:max_len]},
-                'Data': url
-            })
+        update_entry_context_url(ec_url, url, field_name='Resolutions', field_value=resolution[:max_len])
```
Comment (ronykoz) [inline Packs/VirusTotal-Private_API/Integrations/VirusTotal-Private_API/VirusTotal-Private_API.py:725]: why are we trimming the resolution

**Review by ronykoz — APPROVED** [review]

File: Packs/VirusTotal-Private_API/Integrations/VirusTotal-Private_API/VirusTotal-Private_API.py (hunk 1 above)
Comment (roysagi) [reply Packs/VirusTotal-Private_API/Integrations/VirusTotal-Private_API/VirusTotal-Private_API.py:725]: That's how it was before my fix I simply made the code more readable.
Looking at the code my best guess is that its doe for readability - @EdenOhana maybe you can shed some light here?

File: Packs/VirusTotal-Private_API/Integrations/VirusTotal-Private_API/VirusTotal-Private_API.yml [hunk 2]
```diff
@@ -387,23 +422,33 @@ script:
     - contextPath: URL.VirusTotal.DroppedFiles
       description: Downloaded file hashes from this URL
       type: string
+    - contextPath: URL.VirusTotal.ResponseContentSHA256
+      description: SHA 256 of the response content
```
Comment (kirbles19) [inline Packs/VirusTotal-Private_API/Integrations/VirusTotal-Private_API/VirusTotal-Private_API.yml:426]: SHA256 hash of the response content.

File: Packs/VirusTotal-Private_API/Integrations/VirusTotal-Private_API/README.md [hunk 3]
```diff
@@ -2482,6 +2482,16 @@
 <td style="width: 386px;">Resolve date of IPs that resolved to this URL</td>
 </tr>
 <tr>
+<td style="width: 294px;">URL.VirusTotal.ResponseContentSHA256</td>
+<td style="width: 60px;">Unknown</td>
+<td style="width: 386px;">SHA 256 of the response content</td>
```
Comment (kirbles19) [inline Packs/VirusTotal-Private_API/Integrations/VirusTotal-Private_API/README.md:2487]: SHA256 hash of the response content.

**Review by kirbles19 — APPROVED** [review]

**Comment by kirbles19** [comment]
@roysagi Done.

**Pull request merged**